src/markdown-pages/terraform/terragrunt-configuration.mdx

```diff
@@ -0,0 +1,578 @@
+
+[Terragrunt](https://terragrunt.gruntwork.io/#:~:text=Terragrunt%20is%20a%20thin%20wrapper,modules%2C%20and%20managing%20remote%20state.) is a thin wrapper that provides extra tools for keeping your configurations DRY, working with multiple Terraform modules, and managing remote state.
+
+In this guide, you will use Terragrunt to generate your Terraform configurations, create files with Terragrunt, and reuse the Terragrunt configuration to manage multiple environments.
+
+<Video id="ivbp4e8t1f" type="wistia" />
+
+</Intro>
+
+## Before you begin
+
+To use this guide, you should have some basic knowledge of both New Relic and Terraform. If you haven't deployed a New Relic open source agent yet, [install New Relic](https://docs.newrelic.com/docs/agents/manage-apm-agents/installation/install-agent) for your application. Also, [install the Terraform CLI](https://www.terraform.io/intro/getting-started/install.html).
+
+Start by initializing a working directory:
+
+You will also need to install Terragrunt to follow this guide:
+```bash
+brew install terragrunt
+```
+
+To follow the examples in this gude, the accompanying example code is available on [GitHub](https://github.com/jsbnr/nr-terragrunt-intro-example)
+
+```bash
+git clone https://github.com/jsbnr/nr-terraform-intro-example.git
+```
+
+
+<Steps>
+
+<!-- TTERRAGRUNT CONFIGURATION -->
+
+<Step>
+
+## Create a new configuration
+
+Terragrunt is a wrapper that provides extra tooling for your Terraform configurations. In the next steps, you will create a new configuration using Terragrunt, Terraform, and New Relic.
+
+Start by initializing a working directory:
+```bash
+mkdir terragrunt-config && cd terragrunt-config
+```
+In the new directory, create a `terragrunt.hcl` file, create a  ./src directory, and create a `./evironments` directory:
+```bash
+touch terragrunt.hcl
+mkdir src
+mkdir environments
+```
+Inside of your __environments__ folder, create a folder named __dev.__
+In the `./src` directory, create a __main.tf__ and __provider.tf__ fiiles:
+```bash
+mkdir environments/dev
+touch src/main.tf
+touch src/provider.tf
+```
+
+In the __provider.tf__ file add the appropriate Terraform and New Relic provider configuration information:
+```hcl
+terraform {
+    required_version = "~> 0.14.3"
+    required_providers{
+        newrelic = {
+		source = "newrelic/newrelic"
+		version = "~>2.14.0"
+	}
+    }
+}
+```
+
+In the __main.tf__ add a New Relic alert policy named DemoPolicy:
+```hcl
+resource "newrelic_alert_policy" "DemoPolicy" {
+	name = "My Demo Policy"
+}
+```
+
+In the `./enviromments/dev` directory create a new file named `terragrunt.hcl`
+```bash
+cd ./enviroments/dev/
+touch terragrunt.hcl
+```
+
+Inside the dev folder __terragrunt.hcl__ add an include statement. The include statement tells Terragrunt to use any `.hcl` confgiuration files it finds in parent folders.
```

Comment:
This is worded a little funky. Should there be a qualifier before `terragrunt.hcl`? Or maybe change the wording from `Inside the dev folder` to `Change the file`?

Reply:
How about : "Within the dev folder update the `terragrunt.hcl` file by adding the following include statement."